update.py

```diff
@@ -23,6 +28,27 @@
     'NE': 'n'
 }
 
+def get_overrides():
+    filename = "csv/overrides.csv"
+    print(f"Get overrides from GSheet to {filename}")
+    try:
+        sheet2csv.sheet2csv(id=SHEET_OVERRIDES, range=RANGE_OVERRIDES, api_key=GOOGLE_API_KEY, filename=filename)
+    except Exception as e:
+        print("Failed to import {}".format(filename))
+        raise e
+
+def append_overrides():
+    doctors = pd.read_csv('csv/doctors.csv', index_col=['doctor','type'])
```

Comment:
So far we used also id_inst in the index, so this may not be enough.

Reply:
really? This would be unfortunate as id_inst is not persistent...

Reply:
yep, we have `zzzsSt`, but not for most of the rows in institutions, but not yet all - #1

Reply:
also note, that just name + type is not unique for all rows, namely doctors in ŽELEZNIŠKI ZDRAVSTVENI DOM LJUBLJANA, so dropping the inst_id from the index may actually fix #2 (or require fixing #2 first).

Reply:
when dropping the `id` field in #6 i didn't want to change anything else

Reply:
Solved with #9